### PR TITLE
make locale key configurable

### PR DESCRIPTION
**Pull request by grawk**
allows custom locale location

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+var localeContext = 'locale';
 module.exports = {
     languagePackPath: languagePackPath,
     middleware: function(options) {
         var nosuffix = options && options.nosuffix;
+        localeContext = options.localeContext || localeContext;
         return function (req, res, next) {
             if (!res.locals.makara) res.locals.makara = {};
-            res.locals.makara.languagePackPath = languagePackPath(res.locals.locale, nosuffix);
+            res.locals.makara.languagePackPath = languagePackPath(res.locals[localeContext], nosuffix);
```
Comment (gabrielcsapo) [inline index.js:12]: 👍

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 
 node_js:
-  - "0.12"
-  - "0.10"
-  - "iojs"
+  - "4"
```
Comment (gabrielcsapo) [inline .travis.yml:4]: woot woot!

**Review by gabrielcsapo — APPROVED** [review]

**Comment by grawk** [comment]
@aredridel would you please give `grawk` npm publish permission on this package?

**Comment by aredridel** [comment]
Done!

**Pull request merged**